platform/admin/core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/ConfigurationAdmin.java

```diff
@@ -192,7 +201,7 @@ public void setModuleList(List<AdminModule> moduleList) {
         return modules;
     }
 
-    private String getDefaultFactoryLdapFilter() {
+    String getDefaultFactoryLdapFilter() {
```

Comment:
Probably changed them for a unit test and didn't end up needing them, will revert. Good catch.